File: FastAPI/models/product.py
# models/product.py
from pydantic import BaseModel

class ProductCreate(BaseModel):
    id_categorie: int
    id_measure_prod: int
    product_name: str
    unit_value: float
    quantity: float
    description: str
    location: str
    image: str

class ProductUpdate(BaseModel):
    id_categorie: int
    id_measure_prod: int
    product_name: str
    unit_value: float
    quantity: float
    description: str
    location: str
    image: str

class Product(BaseModel):
    id_product: int
    id_categorie: int
    id_measure_prod: int
    product_name: str
    unit_value: float
    quantity: float
    description: str
    location: str
    image: str

File: TierraVivaAndroid/app/src/main/java/com/example/tierravivamarketplace/io/response/Product.kt
package com.example.tierravivamarketplace.io.response

data class Product(
    val id_product: Int,
    val id_categorie: Int,
    val id_measure_prod: Int,
    val product_name: String,
    val unit_value: Float,
    val quantity: Float,
    val description: String,
    val location: String,
    val image: String
)
